lfs-resources/variants/src/main/frontend/src/variantFilesContainer.jsx

```diff
@@ -911,48 +911,64 @@ export default function VariantFilesContainer() {
         </IconButton>
       </DialogTitle>
       <DialogContent className={classes.dialogContent}>
-        <MaterialTable
+        <MaterialReactTable
           data={fileSelected?.sameFiles}
-          style={{ boxShadow : 'none' }}
-          options={{
-            toolbar: false,
-            rowStyle: {
+          enableColumnActions={false}
+          enableColumnFilters={false}
+          enableSorting={false}
+          enableTopToolbar={false}
+          muiTableBodyRowProps={{
+            sx: {
               verticalAlign: 'top',
-            }
+            },
           }}
-          title={""}
           columns={[
-            { title: 'Created',
-              cellStyle: {
-                paddingLeft: 0,
-                fontWeight: "bold",
-                width: '1%',
-                whiteSpace: 'nowrap',
-              },
-              render: rowData => <Link href={rowData["@path"]} underline="hover">
-                                  {DateTime.fromISO(rowData['jcr:created']).toFormat("yyyy-MM-dd")}
-                                </Link> },
-            { title: 'Uploaded By',
-              cellStyle: {
-                width: '50%',
-                whiteSpace: 'pre-wrap',
-                paddingBottom: "8px",
+            { header: 'Created', size: 10,
+              muiTableBodyCellProps: {
+                sx: (theme) => ({
+                  paddingLeft: theme.spacing(1),
```

Comment:
Still not properly aligned:
![image](https://user-images.githubusercontent.com/651980/233742005-aa63d3ba-0a5f-46af-b53d-4d04dc04f659.png)
```suggestion
                  paddingLeft: theme.spacing(2),
```